src/js/stores/BrandStore.jsx

```diff
@@ -5,9 +5,25 @@ import UrlService from 'util/UrlService';
 import dispatcher from 'dispatcher';
 
 const $ = require('jquery');
-
+const moment = require('moment');
 const url = UrlService.getUrl();
 
+// for the UI
+function formatData(data) {
+  const value = Object.assign({}, data);
```

Comment:
So to clone the object to keep it from clobbering the initial data. Trying to avoid side effects in functions since js is dumb.